### PR TITLE
Add ghorbani-mohammad profile

### PR DESCRIPTION
**Pull request by ghorbani-mohammad**
- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1529"></a>

### CODE REVIEW

**Review by Arushay — APPROVED** [review]

File: public/data/ghorbani-mohammad.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/tech_with_mike",
+        "icon": "twitter"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Started Programming",
+        "date": "December 215",
```
Comment (kumarsonsoff3) [inline public/data/ghorbani-mohammad.json:16]:
```suggestion
        "date": "December 2015",
```

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Please fix the `Year` therein! 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

File: public/data/ghorbani-mohammad.json (hunk 1 above)
Comment (ghorbani-mohammad) [reply public/data/ghorbani-mohammad.json]: Thank you for your consideration!

**Review by kabir0x23 — APPROVED** [review]
Looks good ✅

**Review by kumarsonsoff3 — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant, Maintainable

**Pull request merged**